File: java/412. Fizz Buzz/Solution.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> result = new ArrayList<>();

        for (int i = 1; i < n + 1; i++) {
            result.add(createString(i));
        }

        return result;
    }

    public String createString(int number) {
        StringBuilder sb = new StringBuilder();

        if (isDivisibleBy3(number)) sb.append("Fizz");
        if (isDivisibleBy5(number)) sb.append("Buzz");
        if (!isDivisibleBy3(number) && !isDivisibleBy5(number)) sb.append(number);

        return sb.toString();
    }

    public boolean isDivisibleBy3(int number) {
        return number % 3 == 0;
    }

    public boolean isDivisibleBy5(int number) {
        return number % 5 == 0;
    }
}

File: python/412. Fizz Buzz/solution.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        def createStr(number):
            string = ''
            if number % 3 == 0:
                string += 'Fizz'
            if number % 5 == 0:
                string += 'Buzz'
            if number % 5 != 0 and number % 3 != 0:
                string += str(number)

            return string


        result = []

        for i in range(1, n + 1):
            result.append(createStr(i))

        return result
    